querybook/tests/test_lib/test_query_executor/test_hive_progress_percentage.py

```diff
@@ -0,0 +1,23 @@
+from unittest import TestCase
```

Comment:
Can you move the test case to 
querybook/tests/test_lib/test_query_executor/test_clients/test_hive.py

Reply:
done